### PR TITLE
test: Updated datastore Readme for integration test

### PR DESCRIPTION
**Pull request by royjit**
*Issue #, if available:*

*Description of changes:*

*Check points: (check or cross out if not relevant)*

- [ ] Added new tests to cover change, if needed
- [ ] Build succeeds with all target using Swift Package Manager
- [ ] All unit tests pass
- [ ] All integration tests pass
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [ ] PR title conforms to conventional commit style
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/DefaultAuthIAM/README.md [hunk 1]
```diff
@@ -32,14 +32,109 @@ Copy the content of the schema from `AWSDataStoreCategoryPluginAuthIntegrationTe
...
+    No
+ Please provide a name for your user pool: 
+    <amplifyintegCUP>
+
+ How do you want users to be able to sign in? 
+    Username
+ Do you want to add User Pool Groups? 
+    No
+ Do you want to add an admin queries API? 
+    Yes
```
Comment (diegocstn) [inline AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/DefaultAuthIAM/README.md:60]: do we need admin queries?

File: AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/DefaultAuthIAM/README.md [hunk 2]
```diff
@@ -32,14 +32,109 @@ Copy the content of the schema from `AWSDataStoreCategoryPluginAuthIntegrationTe
...
+ Specify the app's refresh token expiration period (in days): 
+    30
+ Do you want to specify the user attributes this app can read and write? 
+    No
+ Do you want to enable any of the following capabilities?
+    (press Enter with none selected)
+ Do you want to use an OAuth flow? 
+    No
+? Do you want to configure Lambda Triggers for Cognito? 
+    Yes
```
Comment (diegocstn) [inline AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/DefaultAuthIAM/README.md:86]: do we need Lambda triggers?

**Review by harsh62 — APPROVED** [review]

File: AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/README.md [hunk 3]
```diff
@@ -236,9 +236,15 @@ type CustomerOrder @model
 }
 
 ```
-3. `amplify push`
+3. If you are using the latest CLI, update cli.json to include `"useExperimentalPipelinedTransformer": false` to ensure that it will use the v1 transformer.
```
Comment (diegocstn) [inline AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/README.md:239]: we might also need to check the value of `transformerversion` flag

**Review by diegocstn — APPROVED** [review]

File: AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/DefaultAuthIAM/README.md (hunk 2 above)
Comment (royjit) [reply AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/DefaultAuthIAM/README.md:86]: yes, used for auth signup triggers.

File: AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/README.md (hunk 3 above)
Comment (royjit) [reply AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/README.md:239]: It worked for me without setting it.

File: AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/DefaultAuthIAM/README.md (hunk 1 above)
Comment (royjit) [reply AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/DefaultAuthIAM/README.md:60]: No, will remove it and push

**Pull request merged**